### PR TITLE
V4 updates

### PR DESCRIPTION
**Pull request by TacoMonkey11**
I split the commits by group so I hope that helps, not as many this time which was nice

### CODE REVIEW

File: manifests/M/misterslime/fabulousclouds/0.x/0.2.x/0.2.3.yaml [hunk 1]
```diff
@@ -27,12 +29,12 @@
   fileType: jar
   md5: 26ce82b9ea86f1c8c3ba8d4e98867103
   downloadPageUrls:
-    modrinth: https://modrinth.com/mod/fabulousclouds/version/RUQzER41
+    modrinth: ~
```
Comment (thefirethirteen) [inline manifests/M/misterslime/fabulousclouds/0.x/0.2.x/0.2.3.yaml:32]: The link seems to point to the right page, any reason for removing it?

File: manifests/A/architectury/architectury/3.x/3.2.x/3.2.52.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# yaml-language-server: $schema=../../../../../../doc/validation-schemas/4.0/version-manifest.json
+
+- loaders:
+    - fabric
+  minecraftVersions:
+    - "1.18"
+    - "1.18.1"
```
Comment (thefirethirteen) [inline manifests/A/architectury/architectury/3.x/3.2.x/3.2.52.yaml:6]: Wrong order!

File: manifests/C/CaffeineMC/lithium/0.x/0.7.x/0.7.6.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+- loaders:
+    - fabric
+  minecraftVersions:
+    - "1.18.1"
+    - "1.18"
+  environment:
+    server: optional
+    client: optional
+  channel: beta
```
Comment (thefirethirteen) [inline manifests/C/CaffeineMC/lithium/0.x/0.7.x/0.7.6.yaml]: Isn't this on the `release` channel?

File: manifests/C/Chocohead/optifabric/1.x/1.12.x/1.12.10.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
+# yaml-language-server: $schema=../../../../../../doc/validation-schemas/4.0/version-manifest.json
+
+- loaders:
+    - fabric
+  minecraftVersions:
+    - "1.18"
```
Comment (thefirethirteen) [inline manifests/C/Chocohead/optifabric/1.x/1.12.x/1.12.10.yaml:6]: Just wondering, did it really not get updated to 1.18.1 lol

Comment (thefirethirteen) [reply manifests/C/Chocohead/optifabric/1.x/1.12.x/1.12.10.yaml:6]: Or does optifine itself just not have a 1.18.1 release?

**Review by thefirethirteen — CHANGES_REQUESTED** [review]
Needs a bit of improvement

File: manifests/M/misterslime/fabulousclouds/0.x/0.2.x/0.2.3.yaml (hunk 1 above)
Comment (TacoMonkey11) [reply manifests/M/misterslime/fabulousclouds/0.x/0.2.x/0.2.3.yaml:32]: Yes, the modrinth version is no longer updated, as the curse has 1.18. The modrinth page has only 1.17 for the same version

File: manifests/C/Chocohead/optifabric/1.x/1.12.x/1.12.10.yaml (hunk 4 above)
Comment (TacoMonkey11) [reply manifests/C/Chocohead/optifabric/1.x/1.12.x/1.12.10.yaml:6]: it does support it, looks like I missed it when copying over the version

File: manifests/C/CaffeineMC/lithium/0.x/0.7.x/0.7.6.yaml (hunk 3 above)
Comment (TacoMonkey11) [reply manifests/C/CaffeineMC/lithium/0.x/0.7.x/0.7.6.yaml]: my bad, copied from the 0.7.6-rc1

File: manifests/C/Chocohead/optifabric/1.x/1.12.x/1.12.10.yaml (hunk 4 above)
Comment (TacoMonkey11) [reply manifests/C/Chocohead/optifabric/1.x/1.12.x/1.12.10.yaml:6]: as much as I want to mess with optifabric users I guess I will fix it =(

Comment (thefirethirteen) [reply manifests/C/Chocohead/optifabric/1.x/1.12.x/1.12.10.yaml:6]: xD

**Review by thefirethirteen — APPROVED** [review]
Everything should be good now!

**Pull request merged**